README.md

```diff
@@ -223,17 +223,26 @@ var rav = new Ravelin({
      * +promise component is included. Ravelin.Promise contains the default.
      */
     // Promise: window.Promise,
-
     /**
      * @prop {string} [rsaKey] The public key used to encrypt cardholder data.
      */
     // rsaKey: '0|...',
-
     /**
      * @prop {Object} [page] Additional properties to describe the initial page
      * load event. Must be JSON-encodable.
      */
     // page: {section: 'about'}
+    /**
+     * @prop {function} [classifyPaste] Override logic for detecting pasted PANs or sensitive values.
+     * @param {ClipboardEvent} e The paste event
+     * @returns {Object} c
+     * @returns {Boolean} [c.pan] Whether the user pasted into a PAN field. If omitted, we look for the data-rvn-pan attribute.
+     * @returns {Boolean} [c.sensitive] Whether the user pasted into a sensitive field. Prevents the pasted value's shape being shared with Ravelin if true. If omitted, we look for the data-rvn-sensitive attribute.
+     */
+    // classifyPaste: e => {{
+    //  pan: e.target.hasAttribute('data-rvn-pan'),
+    //  sensitive: treeHasAttr(e.target, 'data-rvn-sensitive')
+    // }}
```

Comment:
Nit: this example wouldn't be valid JS. Either of these would work.
```suggestion
    // classifyPaste: e => ({
    //  pan: e.target.hasAttribute('data-rvn-pan'),
    //  sensitive: treeHasAttr(e.target, 'data-rvn-sensitive')
    // })
```

```suggestion
    // classifyPaste: e => {
    //   return {
    //     pan: e.target.hasAttribute('data-rvn-pan'),
    //     sensitive: treeHasAttr(e.target, 'data-rvn-sensitive')
    //  };
    // }
```

Reply:
Oops, thanks!